### PR TITLE
DTSPO-4723 - updating readme

### PR DESCRIPTION
**Pull request by endakelly**
- Pointing sbox to shared image gallery
- moving back to local
- hardcode image version for testing
- Trying latest
- using shared_image instead of version
- removing image_id variable
- Adding shared image version back
- back to shared image
- Trying latest
- test other envs
- hardcode latest as image name
- testing with local
- testing with local
- Adding new provider for sig
- tf fmt
- wrong alias name
- removing quotes from provider
- Removing new provider
- adding new provider
- removing data resource temporarily
- Adding data resource
- moving provider inside module
- adding features field
- terraform fmt
- updating nonprod
- Testing plan with sbox sig read permissions removed
- Testing plan with sbox sig read permissions restored
- Changing image version to 1.0.2
- removing deleted bastions from rbac
- Adding empty module for deleted bastions rbac
- Adding data
- change order
- Removing old names
- Remove duplicate role assignment sbox
- testing empty module on rbac sbox
- testing empty module on rbac nonprod
- testing empty module on rbac prod
- removing empty module
- updating readme with sig instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,4 +68,33 @@ module "mybastion" {
 }
 ```
 
-3) Merge the change and a new bastion will be deployed.
\ No newline at end of file
+3) Merge the change and a new bastion will be deployed.
+
+### Updating image used by bastion vm
+
+The image used by the bastion vm is stored in a shared image gallery called hmcts. The image is generated by the Azure pipeline in hmcts/bastion-packer.
```
Comment (timja) [inline README.md:75]:
```suggestion
The image used by the bastion vm is stored in a shared image gallery called hmcts. The image is generated by the Azure pipeline in https://github.com/hmcts/bastion-packer.
```

File: README.md [hunk 2]
```diff
@@ -68,4 +68,33 @@ module "mybastion" {
...
+
+Upon the pipeline running successfully, you should see a new image version in the shared image gallery.
+
+Once you've done the above, update the 10-locals.tf file for each environment to point to the new image version denoted by image_id:
+
+```dtd
+locals {
+  environment             = "sbox"
+  location                = "uksouth"
+  image_id                = "1.0.2"
```
Comment (timja) [inline README.md:90]: this should be called `image_version` not id

File: README.md [hunk 3]
```diff
@@ -68,4 +68,33 @@ module "mybastion" {
...
+Whenever a new version of the image is needed, e.g. a new software package needs installed, do the following:
+
+1. Update the provision-bastion bash script with the changes needed
+2. Update the ubuntu.pkr.hcl file with a new image version
+
+Upon the pipeline running successfully, you should see a new image version in the shared image gallery.
+
+Once you've done the above, update the 10-locals.tf file for each environment to point to the new image version denoted by image_id:
+
+```dtd
```
Comment (timja) [inline README.md:86]:
```suggestion
```hcl
```

File: README.md [hunk 4]
```diff
@@ -68,4 +68,33 @@ module "mybastion" {
...
+The image used by the bastion vm is stored in a shared image gallery called hmcts. The image is generated by the Azure pipeline in hmcts/bastion-packer.
+
+Whenever a new version of the image is needed, e.g. a new software package needs installed, do the following:
+
+1. Update the provision-bastion bash script with the changes needed
+2. Update the ubuntu.pkr.hcl file with a new image version
+
+Upon the pipeline running successfully, you should see a new image version in the shared image gallery.
+
+Once you've done the above, update the 10-locals.tf file for each environment to point to the new image version denoted by image_id:
```
Comment (timja) [inline README.md:84]:
```suggestion
Once you've done the above, update the `10-locals.tf` file for each environment to point to the new image version denoted by image_id:
```

File: README.md [hunk 5]
```diff
@@ -68,4 +68,33 @@ module "mybastion" {
...
+3) Merge the change and a new bastion will be deployed.
+
+### Updating image used by bastion vm
+
+The image used by the bastion vm is stored in a shared image gallery called hmcts. The image is generated by the Azure pipeline in hmcts/bastion-packer.
+
+Whenever a new version of the image is needed, e.g. a new software package needs installed, do the following:
+
+1. Update the provision-bastion bash script with the changes needed
+2. Update the ubuntu.pkr.hcl file with a new image version
```
Comment (timja) [inline README.md:80]: link these files

**Review by timja — APPROVED** [review]
looks good, some suggestions for improvements

**Review by timja — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (endakelly) [reply README.md:90]: Updated

**Pull request merged**